content/rekor-bigquery-dataset.md

```diff
@@ -0,0 +1,24 @@
++++
+title = "Announcing the Sigstore Transparency Log Research Dataset"
+date = "2025-08-15"
+tags = ["sigstore","rekor","transparency-log"]
+draft = false
+author = "Hayden Blauzvern & Eve Martin-Jones, Google Open Source Security Team"
+type = "post"
++++
+
+We're pleased to announce the creation of a new [BigQuery public dataset](https://console.cloud.google.com/marketplace/product/bigquery-public-data/rekor), `rekor`.
+The `rekor` dataset is an easily-queryable mirror of the public good instance of Sigstore's transparency log, Rekor.
+
+As a reminder, signing events are recorded in Rekor, Sigstore's append-only transparency log.
+Software consumers rely on cryptographic proofs of log inclusion to verify that software artifacts are recorded to the log.
+Software producers can verify metadata in the log, verifying that the recorded signature metadata was produced as expected
+when their identities or keys were used to sign artifacts.
```

Comment:
I wonder if mentioning that the opposite is important: while the bigquery dataset is useful for researchers, the bespoke apis are better if you are trying to build something *secure*.

Reply:
Thanks, I mentioned the need for monitoring and why researchers would prefer this dataset instead